app/models/results/web_response_parser.rb

```diff
@@ -104,5 +105,9 @@ def inst_num(type, tree_parent)
         1
       end
     end
+
+    def invalid?(questioning_id)
```

Comment:
Maybe make the method name more specific since this could be confused with validation.